hack/local-cmo.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 set -eu -o pipefail
+set -x
+set -v
```

Comment:
-x is too verbose IMHO and hinders readability.

Reply:
Yep, it was for test purposes only